### PR TITLE
Update meteor.md

### PR DESCRIPTION
**Pull request by xavcz**
Related to https://github.com/apollographql/meteor-integration/issues/77:
- don't "force" folks to use `isomorphic-fetch` by default
- specify that if you want to do server-side rendering, you should use still polyfill server-side `fetch`.

Related to `apollo-client@1.0.0-rc.0` and `apply(Batch)Middleware` (https://github.com/apollographql/meteor-integration/commit/2cf16dd768432083aab22b64b1cd86d2b30c0481#commitcomment-21102646): 
- use a `NetworkInterface` & not a `BatchingInterface` by default. A batching interface when the server actually batches queries, which happen if `dataloader` is setup for instance. As most of the user won't setup `dataloader` by default, just let's not use it by default.

cc @lorensr

### CODE REVIEW

**Comment by lorensr** [comment]
We still have the first universal reason to use batching by default?

image

https://dev-blog.apollodata.com/query-batching-in-apollo-63acfd859862#.jvinqsc21

File: source/meteor.md [hunk 1]
```diff
@@ -217,14 +219,15 @@ import { ApolloProvider } from 'react-apollo';
 import { renderToStringWithData } from 'react-apollo';
 import { match, RouterContext } from 'react-router';
 import Express from 'express';
+// #1 import isomorphic-fetch so the Apollo Client can be created
```
Comment (stubailo) [inline source/meteor.md]: Technically, so that the network interface can be created.

File: source/meteor.md [hunk 2]
```diff
@@ -199,8 +199,10 @@ Meteor.logout(function() {
 
 ## SSR
 There are two additional configurations that you need to keep in mind when using [React Server Side Rendering](http://dev.apollodata.com/react/server-side-rendering.html) with Meteor.
-1. Connect your express server to Meteor's existing server with [WebApp.connectHandlers.use](https://docs.meteor.com/packages/webapp.html)
-2. Do not end the connection with `res.send()` and `res.end()` use `req.dynamicBody` and `req.dynamicHead` instead and call `next()`. [more info](https://github.com/meteor/meteor/pull/3860)
+
+1. Use `isomorphic-fetch` to polyfill `fetch` server-side (used by Apollo Client).
+2. Connect your express server to Meteor's existing server with [WebApp.connectHandlers.use](https://docs.meteor.com/packages/webapp.html)
+3. Do not end the connection with `res.send()` and `res.end()` use `req.dynamicBody` and `req.dynamicHead` instead and call `next()`. [more info](https://github.com/meteor/meteor/pull/3860)
```
Comment (stubailo) [inline source/meteor.md]: Ah nice! Are there any official docs about this? Seems really useful.

**Comment by stubailo** [comment]
I do think everyone should be setting up dataloader - you definitely can run into some pretty bad situations if you don't use it.

**Comment by xavcz** [comment]
Thanks for your comments, it makes sense! 💯 

I've added a note about query batching to redirect to existing resources and let's encourage people to use query batching (totally agreed with "_you definitely can run into some pretty bad situations if you don't use it._" 😄)
➡️ I've updated the integration to use again the batching interface by default.

Also changed the wording in the SSR part to be more precise about network interface & `fetch` 👌 

Considering `dynamicBody/Head`, I didn't find anything official about it. I can go for a PR on the guide/docs for that: do you believe it's more suited on the [`webapp` docs](http://docs.meteor.com/packages/webapp.html)? Or on [routing section in the guide](https://guide.meteor.com/routing.html#server-side)?

**Comment by helfer** [comment]
Thanks @xavcz!

**Pull request merged**